pandas/_libs/tslibs/__init__.py

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python
```

Comment:
why are you adding this? this is NOT a package by definition.

Reply:
yes pls don't add shebangs anywhere.

Reply:
generally you should be adding ``__init__.pyx`` for cython packages (IIRC you can then still import)

Reply:
Will change.

Reply:
> generally you should be adding __init__.pyx for cython packages (IIRC you can then still import)
 
Just renaming led to `ImportError`.  Does something need to go in setup.py for this to work?